lib/cli/file-set-pipeline/log.js

```diff
@@ -12,52 +12,43 @@
  * Dependencies.
  */
 
-var charm = require('charm')(process.stdout);
 var chalk = require('chalk');
-var logUpdate = require('log-update');
 var report = require('vfile-reporter');
 
 /**
+ * Check whether `file` is provided by the user.
+ *
+ * @param {File} file - Virtual file.
+ * @return {boolean?} - whether `file` is provided by the
+ *   user.
+ */
+function isProvidedByUser(file) {
```

Comment:
Don't wanna nitpick, but is extracting this function out really necessary? It's a tiny function that was doing alright as an anonymous function. And this change is not really relevant to the watch PR. It will only confuse our commit diff at the end, in my opinion